ras_party/views/account_view.py

```diff
@@ -13,6 +13,13 @@
 auth = HTTPBasicAuth()
 
 
+@account_view.before_request
```

Comment:
Is there a reason to do this for each blueprint rather than just a general `@app.before_request` for all incoming requests?

Reply:
Not all blueprints use @auth.login_required before request